.circleci/config.yml

```
@@ -12,3 +12,28 @@ workflows:
             # Trigger job also on git tag.
             tags:
               only: /^v.*/
+
+      - architect/push-to-app-catalog:
+          context: "architect"
+          name: push-apptest-app-to-control-plane-catalog
+          app_catalog: "control-plane-catalog"
+          app_catalog_test: "control-plane-test-catalog"
+          chart: "apptest-app"
+          # Needed to trigger job also on git tag.
+          filters:
+            tags:
+              only: /^v.*/
```

It doesn't need to work on tags. 
```suggestion
              ignore: /^.*/
```